cupyx/_ufunc_config.py

```diff
@@ -88,6 +119,23 @@ def seterr(*, divide=None, over=None, under=None, invalid=None, linalg=None):
     return old_state
 
 
+def seterrcall(fallback_mode=None):
```

Comment:
Keep compatibility with NumPy.
```suggestion
def seterrcall(func):
```